Tests/HummingbirdTests/FileMiddlewareTests.swift

```diff
@@ -513,6 +513,35 @@ final class FileMiddlewareTests: XCTestCase {
         }
     }
 
+    func testFilesWithNonLowercaseFileExtensions() async throws {
+        let router = Router()
+        router.middlewares.add(FileMiddleware("."))
+        let app = Application(responder: router.buildResponder())
+
+        let testedExtensions: [(String, UInt)] = [
+            ("jpg", #line),
+            ("JPG", #line),
+            ("JpG", #line),
+            ("JPeG", #line),
+            ("JPEG", #line),
+        ]
+
+        for (index, (testedExtension, line)) in testedExtensions.enumerated() {
+            let fileURL = URL(fileURLWithPath: "\(#function)-\(index)")
+                .appendingPathExtension(testedExtension)
+            let filename = fileURL.lastPathComponent
+            let data = Data()
+            XCTAssertNoThrow(try data.write(to: fileURL))
+            defer { XCTAssertNoThrow(try FileManager.default.removeItem(at: fileURL)) }
+
+            try await app.test(.router) { client in
```

Comment:
Instead of starting the app up five times, why don't you stick the for loop inside the `app.test()` call